pkg/imexport/export.go

```diff
@@ -50,8 +54,93 @@ func createDir(name string, tw *tar.Writer, dir *vfs.DirDoc) error {
 	return err
 }
 
-func export(tw *tar.Writer, fs vfs.VFS) error {
-	root := "/Documents"
+func metadata(tw *tar.Writer, fs vfs.VFS, domain string) error {
+	db := couchdb.SimpleDatabasePrefix(domain)
+	doctype := "io.cozy.files"
+	req := &couchdb.AllDocsRequest{}
+	var results []map[string]interface{}
+	if err := couchdb.GetAllDocs(db, doctype, req, &results); err != nil {
+		return err
+	}
+
+	metaDir := "Metadata/"
+	hdrDir := &tar.Header{
+		Name:     metaDir,
+		Mode:     0755,
+		Typeflag: tar.TypeDir,
+	}
+	if err := tw.WriteHeader(hdrDir); err != nil {
+		return err
+	}
+
+	hdr := &tar.Header{
+		Name:       metaDir + "album.json",
+		Size:       int64(0),
```

Comment:
You can remove this line, 0 is the already the default, and you set the value below